src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -16,6 +16,8 @@
     "41": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "42": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "5": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "48": "0x0D2068Bbe4e3975adc8D6701234fBdA115CdAc19",
+    "49": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
```

Comment:
Did you deploy the contracts on testnet?
```curl --location --request POST 'https://rpc.pioneer.etm.network' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc": "2.0",
    "method": "eth_getCode",
    "params": [
        "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
        "latest"
    ],
    "id": 1}'
```
If I ask for the code to the RPC returns empty
```
{"jsonrpc":"2.0","id":1,"result":"0x"}% 
```